src/codegen/generators/expression_generator.rs

```diff
@@ -2670,3 +2828,31 @@ fn int_value_multiply_accumulate<'ink>(
     }
     llvm.builder.build_load(accum, "accessor").into_int_value()
 }
+
+/// Returns false if any argument in the given list is an (output-)assignment and true otherwise
+fn arguments_are_implicit(arguments: &[&AstNode]) -> bool {
+    !arguments.iter().any(|argument| argument.is_assignment() || argument.is_output_assignment())
+}
```

Comment:
this might be problematic for [#668](https://github.com/PLC-lang/rusty/issues/668), if we decide to allow it

Reply:
I've added a comment referencing https://github.com/PLC-lang/rusty/issues/668